readthedocs/proxito/views/serve.py

```diff
@@ -557,7 +543,9 @@ class ServeRobotsTXT(SettingsOverrideObject):
 
 class ServeSitemapXMLBase(View):
 
-    @method_decorator(cache_page(60 * 60 * 12))  # 12 hours
+    # Always cache this view, since it's the same for all users.
+    cache_response = True
```

Comment:
It was a little confusing to me that we are sometimes using this variable as a class-variable and some other times we are declaring/changing inside a method, making it an instance-variable.

I think this is fine, but we could expand the original comment in the CDN mixin explaining this in a better way.